### ISSUE TITLE
Can we get the source code?

### ISSUE THREAD

**Issue by giantpune** [OhGodACompany/OhGodAnETHlargementPill#3]
Looks like the linux binary has pciutils statically linked in, which make the binary GPL.  Is there any chance you can post the source code to go with that binary?

Please and thank you.

**Comment by michalss** [comment]
yes this is what i was keep asking, it suppose to be a open source, but i would not be surprise if this is another crap...

**Comment by tonymorella** [comment]
I get it with the GPL but come on they did all the work and give this away for free. I am sure they will take out any GPL code links if they are there, work like this is not going to be 100% free they need to get compensated for this work.

**Comment by giantpune** [comment]
@tonymorella 
`but come on they did all the work`  <-- not quite.  They did some of the work.  And somebody else did some of the work and licensed that code under GPL.

**Comment by tonymorella** [comment]
OK let me rephrase that did all the work and figured out how to get the cards to hash faster, the pciutils itself is not going to do this. Yes, I agree they can not statically link to GPL work and not give the credit and link to the open source code plus any changes they made to the GPL code itself, this does not mean they need to open source there code.

**Comment by OhGodAGirl** [comment]
No, the source won't be released.

I'm very sorry you feel entitled to source for free utilities - we could have slapped a development fee on this, but we didn't. It's not _supposed_ to be open source. If you have a problem using it, please, kindly bugger off. 

Also, if you don't like static links, we can remove the Linux version. Would you like to be the person known to be responsible for removing the Linux version, @giantpune?

**Comment by giantpune** [comment]
I'm very sorry you feel entitled to violate software license for code somebody was nice enough to let you use for free.  You do what out want with your project, but don't get your panties in a bunch he's somebody calls you out for breaking the rules.

**Comment by giantpune** [comment]
If you static link to GPL'd code, that makes your whole project GPL.

**Comment by OhGodAGirl** [comment]
@giantpune Oh wow. You mean, the tool we're nice enough to let a bunch of people use for free, and get money from it....using code somebody was nice enough to let us use, for free...almost like it's a chain of freebies. 

Seems I should just package it up and trade it for 1 BTC on the shady undergrounds of bitcointalk.

**Comment by OhGodAnAI** [comment]
I don't think so. You can have part of the code under MIT and a clear section for GPL. 
https://opensource.stackexchange.com/questions/1640/if-im-using-a-gpl-3-library-in-my-project-can-i-license-my-project-under-mit-l

All the Modified GPL code need to remain in GPL, but clear and unrelated portion can be MIT, and then put together, as it is the case here.

If we are interpreting it wrong, you are welcome to seek legal consulting, but i will not comment on this topic further.

**Comment by giantpune** [comment]
The FAQ is very clear about it.

```
If a library is released under the GPL (not the LGPL), does that mean that any software which uses it has to be under the GPL or a GPL-compatible license? (#IfLibraryIsGPL)
Yes, because the program actually links to the library. As such, the terms of the GPL apply to the entire combination. The software modules that link with the library may be under various GPL compatible licenses, but the work as a whole must be licensed under the GPL. See also: What does it mean to say a license is “compatible with the GPL”?
```

Your stackoverflow link is for source code, not the compiled binary.  If you were to release your source code, the parts you wrote could be MIT or whatever license.

**Comment by tpruvot** [comment]
go spam other mining devs (like you?) who put dev fees without sources
